src/Command/IndexFileCommand.php

```diff
@@ -211,7 +213,7 @@ private function migrate(array &$rawData, OutputInterface $output): bool
                 if (null === $file && isset($rawData['content'])) {
                     $fileContent = \base64_decode($rawData['content']);
 
-                    if (\sha1($fileContent) === $rawData[EmsFields::CONTENT_FILE_HASH_FIELD] ?? null) {
```

Comment:
The sha1 or **EmsFields::CONTENT_FILE_HASH_FIELD** is allways set at this point because it's inside a if that is checking it with isset.

```php
   /**
     * @param array<mixed> $rawData
     */
    private function migrate(array &$rawData, OutputInterface $output): bool
    {
        $updated = false;
        if (!empty($rawData)) {
            if (isset($rawData['sha1'])) {
                $file = $this->fileService->getFile($rawData['sha1']);

                if (null === $file && isset($rawData['content'])) {
                    $fileContent = \base64_decode($rawData['content']);

                    if (\sha1($fileContent) === $rawData[EmsFields::CONTENT_FILE_HASH_FIELD]) {
```

Reply:
maybe we should replace 'sha1' with EmsFields::CONTENT_FILE_HASH_FIELD ?

Reply:
Leave it like this that all migrate function can be flagged as deprecated. This when the file were ingested with an elaticsearch's tika plugin.